openapi-generator/lib/src/openapi_generator_runner.dart

```diff
@@ -328,6 +328,7 @@ class OpenapiGenerator extends GeneratorForAnnotation<annots.Openapi> {
     return value.toStringValue() ??
         value.toBoolValue()?.toString() ??
         value.toIntValue()?.toString() ??
+        value.getField('_name')?.toStringValue() ??
```

Comment:
Here we pull out the enum name - https://github.com/dart-lang/sdk/blob/18f37dd8f3db6486f785b2c42a48dfa82de0948b/sdk/lib/core/enum.dart#L47